projects/tiles/src/assets/locales/resources_en_US.json

```
@@ -1,15 +1,23 @@
 {
+  "skyux_tile_expand": {
+    "_description": "Label for expand button in the tile",
+    "message": "Expand the {0} tile"
+  },
   "skyux_tile_help": {
     "_description": "Label for help button in the tile",
-    "message": "Help"
+    "message": "{0} tile help"
   },
   "skyux_tile_settings": {
     "_description": "Label for settings button in the tile",
-    "message": "Settings"
+    "message": "{0} tile settings"
   },
   "skyux_tile_grab_handle": {
     "_description": "Label for grab handle in the tile",
-    "message": "Use arrow keys to reorder. Press up and down arrow keys to change position in column. Press left and right arrow keys to change position to another column."
+    "message": "Move the {0} tile"
+  },
+  "skyux_tile_grab_handle_instructions": {
+    "_description": "Long description for grab handle in the tile",
+    "message": "When focused on a tile's move button, use arrow keys to reorder. Press up and down arrow keys to change position in column. Press left and right arrow keys to change position to another column."
```

```suggestion
    "message": "When focus is placed on a tile's grab handle, use the arrow keys to move it. The up and down keys change its position within a column. The left and right keys move it between columns."
```
Since we use `grab handle` in the two preceding descriptions. I'd use the same terminology in this message to ensure clarity. (Note: If `move button` is actually preferable to `grab handle`, then I'd reverse this suggestion and update those previous instances instead. But FWIW, `grab handle` feels more intuitive to me for what to call this.) ... The other changes are just an attempt to make this flow better.